app/sentry.ts

```diff
@@ -0,0 +1,26 @@
+import * as Sentry from '@sentry/browser';
+import { Ember } from '@sentry/integrations';
+import config from 'open-event-frontend/config/environment';
+
+Sentry.init({
+    integrations: [new Ember()],
+    beforeSend(event: Sentry.Event) {
+        const exception = event.exception?.values?.[0];
+        const errorValue = exception?.value;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 8.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5010664)